ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__metal-4.16-latest-x86.yaml

```diff
@@ -37,7 +37,8 @@ tests:
     test:
     - chain: openshift-qe-installer-bm-ping
     - chain: openshift-qe-installer-bm-deploy
-    - chain: openshift-qe-installer-bm-day2
+    - chain: openshift-qe-installer-bm-day2-label
```

Comment:
just a thought here, are you able to either add multiple refs into a chain hat you can use here to lessen the number of items in the test. Just a note that if one step fails it wont go to the next 
Another idea is to just use `ref: openshift-qe-installer...` since you're not actually having a chain of steps

Reply:
Yeah, we are aware that if one step fails the next won't be executed (its unfortunate that this is not configurable per step).
About the chains, if I understood correctly your comment we are not using them and could we replace them with the following?
```
    test:
    - ref: openshift-qe-installer-bm-ping
    - ref: openshift-qe-installer-bm-deploy
    - ref: openshift-qe-installer-bm-day2
    - ref: openshift-qe-installer-bm-day2-label
```

If that is the case, if you don't mind we can consider merging this PR as is and then opening a new one to change all the **chains** in the BM env to **refs**.